lib/languages/raku.txt

```diff
@@ -0,0 +1,10 @@
+!e
+#`(+\j-->>)+\pj
+#`{+\j-->>}+\pj
+#`[+\j-->>]+\pj
```

Comment:
Since the multi-line/embedded comment supports 4 pairs of characters, and we're already looking for 3 of them, I think we may as well include `<>` too: https://docs.raku.org/language/syntax#Multi-line_/_embedded_comments

Reply:
Good observation. I will add.